xl_auth/commands.py

```diff
@@ -583,11 +583,17 @@ def _get_manually_deleted_permissions():
                 print('Cannot manually deleted permissions for %r on %r; does not exist.'
                       % (email, code))
 
-    # Optionally wipe stray permissions.
+    # Optionally wipe stray permissions, but only if created by admin account.
```

Comment:
Clarify what we mean with 'stray permissions'?

Reply:
* [ ] clarify